apkexporter/apkexporter.go

```diff
@@ -17,7 +18,7 @@ import (
 
 const (
 	passPrefix    = "pass:"
-	fileSchema    = "file://"
+	fileSchema    = "file:/"
```

Comment:
https://tools.ietf.org/html/rfc3986 

based on this after chema always `://` comes 🤔

Reply:
The reason for this is if you have such a file path:`file://path/file.txt` and `file://` is trimmed you are left with `path/file.txt` which is a relative path since it is missing `/` from the beginning. I wanted to parse these paths as absolute so we have the following choices:

1. Ask the users to provide files as `file:///path/file.txt`
2. Trim only `file:/`
3. Trim `file://` and readd "/" at the front.

I think option 1 is not intuitive and error-prone. I've gone with option 2 since I thought there is not much difference to option 3. But giving it a second thought due to your comment, now I think option 3 is better since it protects against users providing `file:/` paths.

I will change the PR accordingly.

Reply:
Ok, so I have read the RFC and it turns out that `file:///` is indeed the correct way to provide absolute paths. So I will go with option 1 from the previous comment.